src/_colors.scss

```diff
@@ -11,15 +11,19 @@ $primary: #304ffd;
 $primary-hover: transparentize($primary, 0.1%);
 $primary-light: lighten($primary, 35%);
 $primary-dark: darken($primary, 15%);
+$primary-shadow: lighten($primary, 30%);
```

Comment:
Shadows are never lighter than original color. I believe it's not shadow, you made poor naming decision